Common/FeatureFlags.swift

```diff
@@ -89,6 +90,13 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.includeServicesInSettingsEnabled = true
         #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
+        #if MOCK_THERAPY_SETTINGS_DISABLED
```

Comment:
Recommend following the pattern used by SIMULATED_CORE_DATA_ENABLED and SCENARIOS_ENABLED because this is basically the same type of flag